File: Logica C/log7.c
#include <stdio.h>

int main() {
    int valores;
    int soma = 0;

    printf("Digite 15 valores inteiros:\n");

    for (int i = 0; i < 15; i++) {
        printf("Valor %d: ", i + 1);
        scanf("%d", &valores);
      if (valores % 2 == 0){
        soma += valores;
      }
    }

    printf("A soma dos valores pares digitados é: %d\n", soma);

    return 0;
}

File: Logica JS/log7.js
function retornaValue(){
    let valores = 0;
    let soma = 0;
    let outputElement = document.getElementById('output');

    outputElement.innerHTML += "Digite 15 valores inteiros: <br>";

    for (let i = 0; i < 15; i++){
        valores = parseInt(prompt("Valor: "+(i+1)));

        if (valores % 2 == 0){
            soma += valores;
        }
    }

    outputElement.innerHTML += "A soma dos valores pares digitados é: "+soma+"<br>";
}

retornaValue()